ui/src/table.tsx

```diff
@@ -376,11 +379,26 @@ const
           filters: c.filterable && m.pagination ? c.filters : undefined,
         }
       })),
+      updateColumns = (filterKey: S, filterVal: S, checked?: B) => {
+        onFilterChange(filterKey, filterVal, checked)
+        setColumns(columns.map(c => {
+          if (columnFilterCount[c.key] || columnFilterCount[c.key] == 0) {
+            let filterCount = ""
+            filterCount = columnFilterCount[c.key] 
+                            ? columnFilterCount[c.key] > 9 
+                              ? " (9+) " 
+                              : " (" + columnFilterCount[c.key] + ") "
+                            : ""
+            c.name = c.name.substring(0, c.key.length) + filterCount
```

Comment:
Instead of adding the filter count to the col name, we would like to have it as a separate square in the top right corner relative to the filter chevron so that it's clear that the number belongs to the filters.